server/aws-lsp-yaml-json/package.json

```diff
@@ -8,10 +8,10 @@
         "test": "ts-mocha -b 'src/**/*.test.ts'"
     },
     "dependencies": {
-        "@aws/language-server-runtimes": "file:../../bin/aws-language-server-runtimes-0.1.1.tgz",
-        "@lsp-placeholder/aws-lsp-core": "^0.0.1",
-        "@lsp-placeholder/aws-lsp-json-common": "^0.0.1",
-        "@lsp-placeholder/aws-lsp-yaml-common": "^0.0.1",
+        "@aws/language-server-runtimes": "^0.x",
+        "@aws/lsp-core": "^0.0.1",
+        "@aws/lsp-json-common": "^0.0.1",
+        "@aws/lsp-yaml-common": "^0.0.1",
```

Comment:
Do we still need these? These aren't published artifacts, right?

Reply:
They are subpackages in this monorepo, and dependencies used to develop this specific server.